enterprise/ha/src/test/java/org/neo4j/kernel/ha/transaction/TransactionThroughMasterSwitchStressIT.java

```diff
@@ -180,7 +180,7 @@ private void reelectTheSameMasterMakingItGoToPendingAndBack( ManagedCluster clus
         cluster.await( memberSeesOtherMemberAsFailed( master, slave ) );
 
         // Fail master and wait for master to go to pending, since cluster lost quorum
-        RepairKit masterRepair = cluster.fail( master, NetworkFlag.IN );
+        RepairKit masterRepair = cluster.fail( master, false, NetworkFlag.IN );
```

Comment:
In a way. In this test, only incoming network is disabled, meaning outgoing heartbeats are received by the cluster and so cluster will never consider the master to be "down". However, since the master isn't receiving heartbeats it will consider itself to be "down". Which the next line waits for.